src/Images.jsx

```diff
@@ -60,14 +60,31 @@ class Images extends React.Component {
                 });
     }
 
+    getUsedByText(image) {
+        const { imageContainerList } = this.props;
+        if (imageContainerList === null) {
+            return { title: _("unused"), count: 0 };
+        }
+        const containers = imageContainerList[image.Id + image.isSystem.toString()];
+        if (containers !== undefined) {
+            const fmt = containers.length === 1 ? _("$0 container") : _("$0 containers");
+            return { title: cockpit.format(fmt, containers.length), count: containers.length };
+        } else {
+            return { title: _("unused"), count: 0 };
+        }
+    }
+
     renderRow(image) {
         const tabs = [];
+        const { title: usedByText, count: usedByCount } = this.getUsedByText(image);
 
         const columns = [
             { title: image.RepoTags ? image.RepoTags[0] : "<none>:<none>", header: true },
+            { title: image.isSystem ? _("system") : <><span className="images-grey-text">user: </span>{this.props.user}</> },
```

Comment:
This needs to be translated. _("user: ")

Reply:
Also this breaks in small screens.
![Screen Shot 2021-09-01 at 17 45 46](https://user-images.githubusercontent.com/14921356/131702597-a48c8697-0aac-4ba9-a382-fd040e3254bb.png)

Reply:
Please avoid translating trailing (or leading) spaces -- it is pretty much garanteed to get messed up during the translation process. Use CSS or HTML structure to do spacing.

Reply:
Added translations without a trailing space, for the small screens I've put the <span> with the {this.props.user} in a <div> so it doesn't break as it does now. I'm not sure if this is the best solution.